frontend/src/metabase/dashboard/reducers.js

```diff
@@ -406,15 +405,6 @@ const loadingDashCards = handleActions(
         };
       },
     },
-    [SET_LOADING_DASHCARDS_COMPLETE]: {
```

Comment:
This was added in https://github.com/metabase/metabase/pull/31578/files#r1240203745, and it makes working on the fix harder, so I had to remove it and addressed the linked comment by dispatching `FETCH_DASHBOARD_CARD_DATA` before `FETCH_CARD_DATA` [here](https://github.com/metabase/metabase/pull/32754/files#diff-636074b28010ac8c8f8597b27dc79a72530c430dbfd344340c399de22a1f6c80R446-R451)